src/main/java/com/seailz/javadiscordwrapper/DiscordJv.java

```diff
@@ -425,4 +428,31 @@ public Cache<Channel> getChannelCache() {
     public void addIntent(@NotNull Intent intent) {
         intents.add(intent);
     }
+
+    private void registerCommand(Command command) {
```

Comment:
You'll want to add checks with the registerCommand method, that all variables meet Discord's requirements.
Add a boolean check to the Checker.java class:

```java
public static void check(boolean check, String message) {
   if (check)
     throw new IllegalArgumentException(message);
}
```

Check for things like, the command name is more than 1 char but less than 32, description is more than 1 char but less than 100, etc.